terraform/config.tf

```diff
@@ -1,5 +1,10 @@
 provider "aws" {
   region = "us-east-1"
```

Comment:
Running `terraform fmt` on this file adds space before the `=` to align it with the one used for the `version` variable assignment.